src/SFA.DAS.ApprenticeCommitments/SFA.DAS.ApprenticeCommitments.csproj

```diff
@@ -9,6 +9,8 @@
     </ItemGroup>
 
     <ItemGroup>
+      <PackageReference Include="FluentValidation" Version="9.5.0" />
+      <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="9.5.0" />
```

Comment:
Is the DI package needed on this project?  I think it may only be required on `SFA.DAS.ApprenticeCommitments.Api`

Reply:
I'll double check, was being lazy